internal/controller/controllers/hypershiftagentserviceconfig_controller.go

```diff
@@ -141,17 +211,23 @@ func (hr *HypershiftAgentServiceConfigReconciler) Reconcile(origCtx context.Cont
 			return result, err
 		}
 	}
+	return ctrl.Result{}, nil
+}
+
+func (hr *HypershiftAgentServiceConfigReconciler) reconcileSpokeComponents(ctx context.Context, log *logrus.Entry, asc ASC) (ctrl.Result, error) {
+	spokeComponents := []component{}
```

Comment:
service account should be part of the RBAC array. It has the same semantics. also is order important here?